### ISSUE TITLE
Make sure MinMaxCloseControl supports FullScreen & Tablet Mode

### ISSUE THREAD

**Issue by zadjii-msft** [microsoft/terminal#2001]
Some thoughts about the Min/Max/Close buttons, and supporting FullScreen or Tablet Mode

Some properties to consider for this TitlebarControl.  This could be also be useful for the Settings UI, if it is presented in a window.
* CanMaximise
* CanMinimise
* CanRestore

Which would hide those buttons if you wanted to make a settings screen which only shows a close button.  Also consider how these TitleBar Controls will handle FullScreen/Tablet Mode

![image](https://user-images.githubusercontent.com/7389110/61311126-a2af5c80-a7ed-11e9-80cd-eab0a88b3a4b.png)
_Full Screen_

![Screenshot (17)](https://user-images.githubusercontent.com/7389110/61311203-ca9ec000-a7ed-11e9-8eea-fd6c32603de2.png)
_Tablet Mode_

_Originally posted by @mdtauk in https://github.com/microsoft/terminal/pull/1948#issuecomment-512048300_

**Comment by AnuthaDev** [comment]
@zadjii-msft I would like to work on it. I believe [this](https://devblogs.microsoft.com/oldnewthing/?p=93815) shall be sufficient to support tablet mode?

**Comment by zadjii-msft** [comment]
@AnuthaDev absolutely, go for it! If @oldnewthing wrote it, then I'm sure it'll work ☺️
